### ISSUE TITLE
put slider numbers in link text for page indicators on home page slider: 2

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#262]
> The page indicators of the slider on the home page currently are reading as “bullet,” this should instead have descriptive text with the slider page number.

![screen shot 2018-04-27 at 2 24 26 pm](https://user-images.githubusercontent.com/263285/39381174-d3f06ada-4a26-11e8-8d55-dca54325dce8.png)

> Links having descriptive text is also useful to the AT user. The text description lets a user distinguish this link from other links in the web page and helps the user determine whether to follow the link.

WCAG 2.0 Conformance: [2.4.4 Link Purpose (In Context)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html): 

> The purpose of each link can be determined from the link text alone or from the link text together with its programmatically determined link context, expect where the purpose of the link would be ambiguous to users in general. (Level A)

Traceability: [Microassist Report 1 > WCAG Compliance > Links > Link Purpose](https://docs.google.com/document/d/10cfUFx-o_kSGBFKCe4MtL45cyc-c-Gwaz3Dxlrp8Qgg/edit#bookmark=id.feh4zw2wft25)

**Comment by paregorios** [comment]
This concern was re-iterated in the Microassist Complex Interaction Plan, as follows:

> The navigation pagers for the carousel currently announce “bullet link” for all of the six bullets, they should have descriptive text explaining the specific bullet link (slide 1, slide 2, etc.). These navigation elements should be buttons.
>
>When selecting the navigation pagers, the focus should move to the content in the active carousel.

The focus requirement appears to me to be new/additional. @skleinfeldt do you think we need to re-estimate this task?

**Comment by cguardia** [comment]
Looked at this one a bit. Currently, we are using bullets as a navigation mechanism, which is what shows up in the links. Changing this for descriptive text would replace the bullets for the text in question, making the links confusing. The slider has the option to use buttons instead, which uses the appropriate page numbers. I would recommend using that option (see slider settings, type of navigation). If this is not possible or desired, we should discuss it further.

**Comment by paregorios** [comment]
@cguardia I concur with your suggestion. Let's switch to buttons.

**Comment by cguardia** [comment]
Made the change on staging. If it looks good I can do it on production.

**Comment by paregorios** [comment]
Great! Yes, please do implement on production.

**Comment by cguardia** [comment]
Done.
